lib/salus/package_utils/gradle_dependency_parser.rb

```
@@ -0,0 +1,30 @@
+module Gradle
+  GRADLE7 = "/opt/gradle/gradle-7.3.3/bin/gradle".freeze
+  GRADLE6 = "/opt/gradle/gradle-6.9.2/bin/gradle".freeze
+
+  def gradle_dependencies
+    dependency_metadata_regex = /-\s(?<group_id>.+):(?<artifact_id>.+):(?<version>.+)/
+
+    # 'gradle dependencies' command needs to be run in the folder where buid.gradle is present.
+    shell_result = run_shell("#{GRADLE7} dependencies")
+
+    shell_result = run_shell("#{GRADLE6} dependencies") if !shell_result.success?
+    if !shell_result.success?
+      report_error("Gradle Version Not supported. Please Upgrade to gradle version 6 and above")
+      return []
+    end
+
+    dependencies = []
+
+    shell_result.stdout.scan(dependency_metadata_regex).each do |dependency_properties|
+      dependency_hash = {}
+      dependency_hash['group_id'] = dependency_properties[0]
```

We should probably report an error here if the dependency_properties.size < 3